crates/runtime-factors/src/lib.rs

```diff
@@ -100,6 +100,10 @@ pub struct TriggerAppArgs {
     /// To run from a file, prefix the filename with @ e.g. spin up --sqlite @migration.sql
     #[clap(long = "sqlite")]
     pub sqlite_statements: Vec<String>,
+
+    // Sets the maxmimum memory allocation limit for an instance.
+    #[clap(long = "max-memory-size", env = "MAX_MEMORY_SIZE")]
+    pub max_memory_size: Option<usize>,
```

Comment:
I wounder if this should be e.g. `max-instance-memory` to make it clear that its per-instance and not the whole server?